test-common/src/testFixtures/kotlin/org/octopusden/octopus/components/registry/test/BaseComponentsRegistryServiceTest.kt

```diff
@@ -84,6 +84,19 @@ val VCS_SETTINGS = VCSSettingsDTO(
     )
 )
 
+val VCS_SETTINGS_HOTFIX = VCSSettingsDTO(
```

Comment:
it's possible to avoid duplication
```
private fun createVcsSettings(version: String) = VCSSettingsDTO(
    versionControlSystemRoots = listOf(
        VersionControlSystemRootDTO(
            vcsPath = "ssh://hg@mercurial/sub",
            name = "main",
            branch = "v2",
            tag = "SUB-$version",
            type = RepositoryType.MERCURIAL,
            hotfixBranch = "hotfix/3.0.0"
        )
    )
)

val VCS_SETTINGS = createVcsSettings(BUILD_VERSION)  // "SUB-3.0.0"
val VCS_SETTINGS_HOTFIX = createVcsSettings(HOTFIX_VERSION)  // "SUB-3.0.0-0"
```